app/templates/subsidy/applications/edit.hbs

```diff
@@ -17,17 +17,35 @@
   </Group>
   <Group class="au-c-toolbar__group--col">
     <AuButtonGroup>
+      <AuDropdown
+        @title="Bestanden Downloaden"
+        @buttonLabel="Account settings"
+        @alignment="right"
+        @skin="secondary"
+        @icon="download"
+        @iconAlignment="left"
+        role="menu"
+        class="au-u-hide-on-print"
+      >
+        <AuButton @skin="link" role="menuitem" {{on "click" this.exportSubsidyAsPDF}}>
+          Download huidige stap (PDF)
+        </AuButton>
+        <AuButton @skin="link" role="menuitem" {{on "click" this.downloadBijlagen}}>
+          Alle bijlagen downloaden in .zip
+        </AuButton>
+      </AuDropdown>
```

Comment:
It feels a bit strange to always show this, even if the form hasn't been submitted yet?

Reply:
Agreed, I'll hide it when the subsidystep is in concept phase